src/Elements/utilities.jl

```diff
@@ -0,0 +1,24 @@
+["Helper functions."]
+
+"""
+Return the value stored for the given key, or the given default value if no mapping for the key is present.
+Search begins with the node ``n`` itself followed by any parent node configuration setting.
+"""
+function getconfig(n::Node, key::Symbol, default::Any=:notfound)
```

Comment:
``::Any`` isn't needed in the signature. Spaces between ``default`` and ``:notfound`` to follow the rest of the codebase.